src/components/Person/Person.jsx

```diff
@@ -1 +1,13 @@
-// export const Person = ({ person }) => ();
+import React from 'react';
+
+export const Person = ({ person }) => {
+  const partner = person.sex === 'm' ? 'wife' : 'husband';
+
+  return (
+    <section className="Person">
+      <h2 className="Person__name">My name is {person.name}</h2>
+      <p className="Person__age">{person.age ? `I am ${person.age}` : null}</p>
```

Comment:
don't render this paragraph (tag `p`) at all if `age` is not defined
use [conditional rendering](https://react.dev/learn/conditional-rendering) instead of ternary with `null` in such cases